docs/usage.rst

```diff
@@ -172,9 +172,32 @@ container based on the given image name. This assumes the ``fv3config`` package
 ``fv3gfs`` python wrapper are installed inside the container, along with any
 dependencies.
 
+
```

Comment:
If you get the chance, remove this extra newline.

Reply:
done